src/playground-code-editor.ts

```diff
@@ -279,18 +290,19 @@ export class PlaygroundCodeEditor extends LitElement {
             break;
           case 'diagnostics':
             this._showDiagnostics();
+            this._showCompletions();
```

Comment:
Why do we need to show completions when diagnostics change?

Reply:
This was a typo from a merge. Fixed in local